src/app/(auth)/account/add-new-user/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { getTranslations } from 'next-intl/server';
+
+import { getOrganizations } from '@/actions/getOrganisations';
+import { getUsersMe } from '@/actions/getUsersMe';
+import { auth } from '@/auth';
+import { AddNewUserForm } from '@/components/account/add-new-user/AddNewUserForm';
+
+export default async function AddNewUserPage(props: {
+  searchParams?: Promise<{
+    organization_id?: string;
+  }>;
+}) {
+  const t = await getTranslations('AddNewUser');
+
+  const session = await auth();
```

Comment:
why not `getSession()`?

Reply:
https://authjs.dev/getting-started/session-management/get-session